Dockerfile

```diff
@@ -65,15 +67,86 @@ ENV LD_LIBRARY_PATH="/opt/Python35/lib:${LD_LIBRARY_PATH}"
 # Install pip packages for Python 3.5.
 RUN /opt/Python35/bin/pip3.5 install six progressbar2==3.37.1 wheel
 
+
+# Download and install Python 3.6.
+RUN source /opt/rh/devtoolset-7/enable && \
```

Comment:
this is always the same for 3.5, 3.6 and 3.7. Maybe a script could be used instead?

Reply:
we decided to leave it as this for now